cli.js

```diff
@@ -411,30 +441,95 @@ function toDecimals(value, decimals, fixed) {
   return value
 }
 
+// List fetched from https://github.com/ethereum-lists/chains/blob/master/_data/chains
+function getExplorerLink() {
+  switch (netId) {
+    case 56:
+      return 'bscscan.com'
+    case 100:
+      return 'blockscout.com/poa/xdai'
+    case 137:
+      return 'polygonscan.com'
+    case 42161:
+      return 'arbiscan.io'
+    case 43114:
+      return 'snowtrace.io'
+    case 5:
+      return 'goerli.etherscan.io'
+    case 42:
+      return 'kovan.etherscan.io'
+    default:
+      return 'etherscan.io'
+  }
+}
+
+// List fetched from https://github.com/trustwallet/assets/tree/master/blockchains
 function getCurrentNetworkName() {
   switch (netId) {
     case 1:
-      return ''
+      return 'Ethereum'
```

Comment:
Fetching network name for `console.log` and the directory of local cache files to support various networks.